modules/compliance-anatomy.adoc

```diff
@@ -0,0 +1,284 @@
+
+      Data
+      ====
+      exit-code:
+      ----
+      2
+      results:
+      ----
+      <?xml version="1.0" encoding="UTF-8"?>
+      ...
+
+----
+
+Scanner pods for `Platform` scans are similar, except:
+
+* There is one extra init container called `api-resource-collector` that reads the OpenSCAP content provided by the content-container init, container, figures out which API resources the content needs to examine and stores those API resources to a shared directory where the `scanner` container would read them from.
+
+* The `scanner` container does not need to mount the host filesystem.
+
+When the scanner pods are done, the scans move on to the Aggregating phase.
+
+=== Aggregating phase
+In the aggregating phase, the scan controller spawns yet another pod called the aggregator pod. Its purpose it to take the result `ConfigMap` objects, read the results and for each check result create the corresponding k8s object. If the check failure can be automatically remediated, a `ComplianceRemediation` object is created. To provide human-readable metadata for the checks and remediations, the aggregator pod also mounts the OpenSCAP content using an init container.
+
+When a `ConfigMap` is processed by an aggregator pod,it is labeled the `compliance-remediations/processed` label. The result of this phase are `ComplianceCheckResult` objects:
+
+[source,terminal]
+----
+$ oc get compliancecheckresults -lcompliance.openshift.io/scan-name=rhcos4-e8-worker
+NAME                                                       STATUS   SEVERITY
+rhcos4-e8-worker-accounts-no-uid-except-zero               PASS     high
+rhcos4-e8-worker-audit-rules-dac-modification-chmod        FAIL     medium
+----
+and `ComplianceRemediation` objects:
+
+[source,terminal]
+----
+$ oc get complianceremediations -lcompliance.openshift.io/scan-name=rhcos4-e8-worker
+NAME                                                       STATE
+rhcos4-e8-worker-audit-rules-dac-modification-chmod        NotApplied
+rhcos4-e8-worker-audit-rules-dac-modification-chown        NotApplied
+rhcos4-e8-worker-audit-rules-execution-chcon               NotApplied
+rhcos4-e8-worker-audit-rules-execution-restorecon          NotApplied
+rhcos4-e8-worker-audit-rules-execution-semanage            NotApplied
+rhcos4-e8-worker-audit-rules-execution-setfiles            NotApplied
+----
+
+After these CRs are created, the aggregator pod exits and the scan moves on to the Done phase.
+
+=== Done phase
+In the final scan phase, the scan resources are cleaned up if needed and the `ResultServer` deployment is either scaled down (if the scan was one-time) or deleted if the scan is continuous; the next scan instance would then recreate the deployment again.
+
+It is also possible to trigger a re-run of a scan in the Done phase by annotating it:
+
+[source,terminal]
+----
+$ oc annotate compliancescans/rhcos4-e8-worker compliance.openshift.io/rescan=
+----
```

Comment:
should be used `$scan_name` to get actual scan name as we don't any reference above mentioned scan name in this section. 
`$ oc annotate compliancescans/$scan_name compliance.openshift.io/rescan=`

Reply:
Fixed this in multiple places. Thanks!